common/src/main/java/be/portal/job/mappers/application/ApplicationMapper.java

```diff
@@ -20,10 +21,12 @@ public interface ApplicationMapper {
     @Mapping(target = "id", ignore= true)
     @Mapping(target = "createdAt", ignore = true)
     @Mapping(target = "updatedAt", ignore = true)
+    @Mapping(target = "applyDate", ignore = false)
```

Comment:
Finally, if the mapper can map this applyDate from params to the applyDate of Application, there is no need to put it, because I think it will map it automatically since they have the same name.